### PR TITLE
add new term ATP1A3-associated neurological disorder

### LINKED ISSUE #3091: ATP1A3- associated neurological disorders
**Preferred gene-related syndrome label**
ATP1A3- associated neurological disorders

**Synonyms**

**Parent term (use [OLS](https://www.ebi.ac.uk/ols/ontologies/mondo), or your favorite ontology browser)**

**Definition**
This is a request for a new term for neurological disorders associated with ATP1A3. This is the nomenclature used in Gene Reviews, which give the following definition: ATP1A3-related neurologic disorders represent a clinical continuum in which at least three distinct phenotypes have been delineated: rapid-onset dystonia-parkinsonism (RDP); alternating hemiplegia of childhood (ACH); and cerebellar ataxia, areflexia, pes cavus, optic atrophy, and sensorineural hearing loss (CAPOS). However, some affected individuals have intermediate phenotypes or only a few features that do not fit well into one of these major phenotypes.

Mondo and OMIM list the same three disease entities in association with ATP1A3:  Dystonia 12, Alternating hemiplegia of childhood 2, and Cerebellar Ataxia, Areflexia, Pes Cavus, Optic Atrophy, And Sensorineural Hearing Loss (CAPOS). Using the ClinGen lumping and splitting guidelines, the ClinGen Syndromic Disorders GCEP determined that these three disease entities should be lumped together as a single entity. All three disease entities share overlapping features that can be considered a spectrum of disease, and there is no genotype-phenotype correlation (the same variants are associated with more than one disorder). 

**Definition source (Please give PubMed ID, if applicable, in format PMID:#######)
PMID: 33762331
PMID: 31178018
PMID: 20301295

**Children terms (if applicable)** Should any existing terms be re-classified as children underneath this new proposed term? 

**Your nano-attribution (ORCID) or URL for a working group**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

### PR DESCRIPTION
**Pull request by sabrinatoro**
Fix #3091

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -435463,6 +435463,18 @@ synonym: "attenuated hypophosphatasia" EXACT [https://www.clinicalgenome.org/aff
 is_a: MONDO:0018570 {source="PMID:32973344", source="https://www.clinicalgenome.org/affiliation/40065/"} ! hypophosphatasia
 property_value: created_by http://orcid.org/0000-0002-5460-8025
 
+[Term]
+id: MONDO:0700002
+name: ATP1A3-associated neurological disorder
+def: "Any neurological disorder in which the cause of the disease is a mutation in the ATP1A3." [MONDO:patterns/disease_series_by_gene]
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:435469]: You can add the PMID references to the definition as well.

**Comment by nicolevasilevsky** [comment]
This looks good @sabrinatoro! Did you run the reasoner and note that there are several terms that autoclassify under this term? This is perfect - these are OMIM terms that have mutations in this same gene.

image

there is a conflict, we can ask @matentzn for help with resolving it.

**Review by nicolevasilevsky — APPROVED** [review]
I approve, but could you make the minor change to add the PMIDs as a dbxref to the definitions
@sabrinatoro :eyes:
(tagging you just in case you missed this)

**Comment by matentzn** [comment]
I fixed the conflict

**Comment by nicolevasilevsky** [comment]
thank you!

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo]: added PMID

**Pull request merged**